src/checkout-buttons/strategies/googlepay/googlepay-button-strategy.ts

```diff
@@ -71,13 +71,17 @@ export default class GooglePayButtonStrategy implements CheckoutButtonStrategy {
 
         return this._googlePayPaymentProcessor.displayWallet()
             .then(paymentData => this._googlePayPaymentProcessor.handleSuccess(paymentData)
-            .then(() => {
-                if (paymentData.shippingAddress) {
-                    this._googlePayPaymentProcessor.updateShippingAddress(paymentData.shippingAddress);
-                }
-            }))
+                .then(() => {
```

Comment:
Might be easier to keep the indentation as it was, but close the brackets on the line before, e.g:
```js
.then(paymentData => this._googlePayPaymentProcessor.handleSuccess(paymentData))
.then(() => {
  if (paymentData.shippingAddress) {
  // ...
```

or if you want to make it even better, switch to async/await

```js
const paymentData = await this._googlePayPaymentProcessor.displayWallet();
await this._googlePayPaymentProcessor.handleSuccess(paymentData);

if (paymentData.shippingAddress) {
  this._googlePayPaymentProcessor.updateShippingAddress(paymentData.shippingAddress);
}

await this._onPaymentSelectComplete();
```